cypress/e2e/specialFeatures/liteSiteWeight/assertions/liteSiteWeight.js

```diff
@@ -0,0 +1,52 @@
+import interceptGetRequests from '../helpers/interceptGetRequests';
+import getPageSizeInKB from '../helpers/getPageSizeInKB';
+
+const MAX_PAGE_WEIGHT = 100;
```

Comment:
Probs worth adding KB to the end of this to make it more clear
```suggestion
const MAX_PAGE_WEIGHT_KB = 100;
```